### PR TITLE
patch issue on autoDeployCheckFile()

### PR DESCRIPTION
**Pull request by paul45**
in autoDeployCheckFile() when $dirMin is used 
`/if (! file_exists($filePath))` use the bad directory

and optimisation on autoDeployCheckFile() for call deployFiles() only one time

### CODE REVIEW

File: src/Minifier.php [hunk 1]
```diff
@@ -232,7 +232,7 @@ protected function autoDeployCheckFile(string $fileType, string $filename): bool
         $dirMin = 'dirMin' . ucfirst(strtolower($fileType));
 
         $assets   = [$filename => $this->config->$fileType[$filename]];
-        $filePath = $this->config->$dir . '/' . $filename;
+        $filePath = $this->config->$dirMin . '/' . $filename;
```
Comment (michalsn) [inline src/Minifier.php:235]: Good catch, but what if we don't use `$dirMin` option? The default value for this is `null`.

File: src/Minifier.php [hunk 2]
```diff
@@ -245,16 +245,20 @@ protected function autoDeployCheckFile(string $fileType, string $filename): bool
...
         // loop though the files and check last update time
+        $needDeploy = false;
         foreach ($assets[$filename] as $file)
         {
             $currentFileTime = filemtime($this->config->$dir . '/' . $file);
             if ($currentFileTime > $lastDeployTime)
             {
-                $this->deployFiles($fileType, $assets, $this->config->$dir, $this->config->$dirMin);
-                return true;
+                $needDeploy = true;
```
Comment (michalsn) [inline src/Minifier.php:254]: How is this an optimization? We have `return` here so if `deployFiles()` needs to be called it will only be called once.

Comment (paul45) [reply src/Minifier.php:254]: ho yes you have right, my bad

**Comment by michalsn** [comment]
Thanks @paul45

**Pull request merged**